projects/plugins/jetpack/extensions/blocks/tiled-gallery/index.js

```diff
@@ -228,6 +228,7 @@ export const settings = {
 		align: [ 'center', 'wide', 'full' ],
 		customClassName: false,
 		html: false,
+		__experimentalExposeControlsToChildren: false,
```

Comment:
This will only impact mobile, given that inner blocks aren't being used by the web version of the block.